client/src/pages/Restaurant/FoodList/FoodCard.js

```diff
@@ -1,5 +1,57 @@
-function FoodCard() {
-  return <div>FoodCard</div>;
+function FoodCard({ name, price, description, imageUrl }) {
+  const classes = {
+    cardContainer: {
+      boxSizing: "border-box",
+      width: "100%",
+      height: "68px",
+      border: "2px solid #5a5a5a",
+      padding: "2px",
+      display: "flex",
+      gap: "0.25rem",
+      backgroundColor: "hsl(35,0%,97%)",
+    },
+    foodInfo: {
+      maxWidth: "740px",
+    },
+    foodName: {
+      fontSize: "1rem",
+      fontWeight: "bold",
+      overflow: "hidden",
+      whitespace: "nowrap",
+      textOverflow: "ellipsis",
+      height: "30%",
```

Comment:
fontSize 1rem not needed

Remove bold
What's the height for?

Reply:
_Height_: It allows the name/title of the food to get cut off/dissapear when resizing to a smaller window. Doesn't work without it